front-end/src/screens/Home/LatestProposal/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { useLatestDemocracyProposalPostsQuery } from '../../../generated/graphql';
+import Proposals from '../../../components/Listings/ProposalsListing';
```

Comment:
idem. Tipp, on VScode, just select the word, press F2, this will change all occurences of the word.